src/components/header/Navbar.tsx

```diff
@@ -4,13 +4,18 @@ import { Link } from 'react-router-dom';
 import { routesForAuthUser, routesForGuest } from '../../constants/navbar';
 import style from './navbar.module.scss';
 import classNames from 'classnames';
+import { useAppDispatch, useAppSelector } from "../../hooks/redux";
+import { userSlice } from "../../redux/reducers/authReducer";
+import storage from '../../utils/storage';
 
 const Navbar: FC = () => {
-  const [isAuthUser, setIsAuthUser] = useState<boolean>(false);
-  const onToggleAuthFlag: () => void = () =>
-    setIsAuthUser((prevStateAuth) => !prevStateAuth);
+  const dispatch = useAppDispatch();
+  const { signInUser } = userSlice.actions;
+  const { user } = useAppSelector(state => state.user)
   const [openBurger, setOpenBurger] = useState<boolean>(false);
 
+  storage.getToken() && dispatch(signInUser());
```

Comment:
Strange construction. Please refactor it.
```suggestion
  if (Boolean(storage.getToken()) {
      dispatch(signInUser());
  }
```